app/main.development.js

```diff
@@ -179,6 +181,14 @@ const launchDCRWallet = () => {
     args.push('--testnet');
   }
 
+  // Add any extra args if defined.
+  if (argv.extrawalletargs != undefined) {
+    var extraArgs = argv.extrawalletargs.split(' ');
```

Comment:
I think people who use paths with spaces in them should be either shunned or punished for that type of behavior but I'll see if I can find a way to accommodate that perversion.